src/components/pages/InstructorHome/index.js

```diff
@@ -1,23 +1,59 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import '../../../styles/InstructorStyles/index.less';
 import InstructorSidebar from './InstructorSidebar';
-import CalendarApp from '../../common/calendar/Calendar';
 import { Layout } from 'antd';
+import CourseCard from '../AdminHome/AdminCourseCard';
+import { useDispatch } from 'react-redux';
+import { getCourses } from '../../../redux/actions/instructorActions';
+import { useOktaAuth } from '@okta/okta-react';
+import { connect } from 'react-redux';
+//import AdminAddCoursesForm from '../AdminHome/AdminAddCoursesForm.js'
+//The above import will be used to add the add courses form, currently being developed in ticket BL-868
 
 const { Content } = Layout;
-const InstructorHome = () => {
+const InstructorHome = props => {
+  const { courses } = props;
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const idToken = useOktaAuth().oktaAuth.getIdToken();
+
+  useEffect(() => {
+    dispatch(getCourses(idToken));
```

Comment:
I think you don't need dispatch here (the import can also be left out) because that's being done in the action file. You'd just have to include the redux action function (`getCourses` in this case) in props and add it to the connect actionCreator. 
Ex. `export default connect(mapStateToProps, {getCourses})(InstructorHome);`

Reply:
I have tried this a few different ways, and I am not able to get it working without the dispatch. I am not entirely sure why, though I could be missing something. I have a lot to learn about Redux. Do you have any thoughts on why it wouldn't work without dispatch, even when it's connected? If I console.log without the dispatch, it logs the function, but doesn't actually give me a response.

Reply:
My guess would be that because the function isn't passed into the action dispatcher, that redux isn't passing it through to props. Don't forget that it will be available via props when done correctly. See the example at the end of my last comment. You'll also have to extract the function from props at the top.